models/intel/face-landmarks-98-detection-0001/accuracy-check.yml

```diff
@@ -0,0 +1,28 @@
+models:
+  - name: face_98_landmarks_detection
+
+    launchers:
+      - framework: dlsdk
+        device: CPU
+        allow_reshape_input: True
```

Comment:
below you resize image to fixed size. I believe you do not need reshape network and this option should be removed